### ISSUE TITLE
Blaze card ignores leading margin for large dynamic type

### ISSUE THREAD

**Issue by twstokes** [wordpress-mobile/WordPress-iOS#20506]
### Expected behavior

Leading edge doesn't have margin or exceeds the bounds of the screen.

### Actual behavior

Leading edge to match the margin of other cards.

<img src="https://user-images.githubusercontent.com/2092798/231252011-6b1384e9-8bac-40b0-a15c-b35bd722277c.png" width="300" />

### Steps to reproduce the behavior
1. Log into a WordPress.com account that supports Blaze
2. Go to My Site -> Home tab
3. Set Dynamic Type to XXX Large or higher
4. Observe the leading margin for the Blaze card

**Comment by kean** [comment]
This was fixed in #20772
Duplicate #20687
